doc/tech/README.md

```diff
@@ -90,23 +93,70 @@ Any specification work created within CNTT **must** conform to set of principles
+
+<p align="center"><img src="./figures/cntt_scope.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 2:</b> Functional Scope of CNTT</p>
+
+<a name="3.2"></a>
+## 3.2 Out of Scope Components
+While the nature of the CNTT might seem quite broad, the following areas are not at this time part of the scope of this effort.  
+- Hardware specifications: beyond the abstracted high-level CPU, memory, network interface and storage elements.  The intention is to write the documents so they are general enough that any vendor hardware can be used in a conformant implementation without making significant changes to the model.
+- Workload specifications: Other than the API interfaces when they directly need to touch the workloads themselves, the intention is to assume the workload application is a blackbox that the cloud infrastructure is providing resources to.  The majority of interactions for lifecycle management of the workloads will be through the cloud infrastructure whenever possible.
+- Company specific requirements: The CNTT documents are designed to be general enough that most operators and others in the Open Source communities will be able to adapt and extend them to their own non-functional requirements.
+
+<a name="3.3"></a>
+## 3.3 Relationship to other industry projects
+The CNTT work is not done in a vacuum.  The intention from the beginning was to utilize the work from other Open Source and standards bodies within the industry. Some of the projects, but by no means all, that are related in some way to the CNTT efforts include:
+
+- ETSI NFV ISG
+- OpenStack
+- OPNFV
+- ONAP
+- CNCF
+- MEF
+- TM Forum
+- OSM (ETSi Open Source MANO project)
+- VMware (While not an Open Source project, VMware is a commonly used platform used for VNF deployments in the telecom industry)
+
+The ETSI NFV ISG is very closely related to the CNTT, in that it is a group that is working on supporting technologies for NFV applications. To facilitate more collaboration as the project matures, the CNTT scope (**Figure 2** above) purposely references certain ETSI NFV reference points, as specified by [ETSI GS NFV 002](https://www.etsi.org/deliver/etsi_gs/NFV/001_099/002/01.02.01_60/gs_NFV002v010201p.pdf).
+
+The CNTT is also closely aligned with OVP, an open source, community-led compliance and verification program that demonstrates the readiness and availability of commercial NFV products and services, including cloud infrastructure and VNFs, using OPNFV. OVP combines open source-based automated compliance and verification testing for multiple parts of the NFV stack specifications established by ONAP, multiple SDOs such as ETSI and GSMA, and the LF Networking End User Advisory Group (EUAG).
```

Comment:
OVP is not mentioned in the above list, should we add it to the list, for example, OPNFV/OVP

Reply:
My understand is that OVP is the verification part of OPNFV and so already covered.  Same as Kubernetes or TUG being part of CNCF.